packages/api/src/utils/warnOnce.js

```diff
@@ -0,0 +1,10 @@
+export default function warnOnce(message) {
```

Comment:
👍

Reply:
Should move all deprecation warnings to this model. 😉